conf/application.conf

```diff
@@ -186,7 +186,7 @@ urls {
   goodsLocationCodesForDataElement = "https://www.gov.uk/government/collections/goods-location-codes-for-data-element-523-of-the-customs-declaration-service"
   tariffCdsChiefSupplement = "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/956071/Tariff_Volume_3_Great_Britain_Supplement_CDS_and_CHIEF_version_5.3__1_.odt"
   notesForMucrConsolidation = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-2-references-of-messages-document-certificates-and-authorisations#de-21-simplified-declaration-previous-documents-box-40-declaration-previous-document"
-  arriveOrDepartExportsService = "https://www.gov.uk"  // To replace. It will be provided through another ticket
+  arriveOrDepartExportsService = "/customs-movements"
```

Comment:
Are you sure this will work? I'd expect it to be in a similar way to the line 161 and then overridden on environments.

Reply:
Ok. I'll use that one, without `/start` btw

Reply:
Yes it works, but I'll rename it as the property at line 161